src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -5,7 +5,8 @@ import { connect } from 'react-redux';
 import Modal from '@department-of-veterans-affairs/component-library/Modal';
 
 import FedWarning from 'applications/login/components/FedWarning';
-import SignInButtons from 'applications/login/components/SignInButtons';
+import NewDesignButtons from 'applications/login/components/NewDesignButtons';
+import OriginalDesignButtons from 'applications/login/components/OriginalDesignButtons';
 import SubmitSignInForm from 'platform/static-data/SubmitSignInForm';
 import SignInDescription from 'applications/login/components/SignInDescription';
```

Comment:
Is there any way we can have these components moved into the `src/platfoprm/user/authentication/components` directory? I think it's preferable to have app code importing from `src/platform` rather than the other way around when possible.

Reply:
^ Sure I can make the updates.